leapp/libraries/stdlib/call.py

```diff
@@ -68,21 +65,19 @@ def _multiplex(ep, read_fds, callback_raw, callback_linebuffered,
                     hupped.add(fd)
                     ep.unregister(fd)
 
-    # FIXME: (**LINE_BUFFER**)
-    # FIXME: Issue #488
-    # # Process leftovers from line buffering
-    # for (fd, lb) in linebufs.items():
-    #     if lb:
-    #         # [stdout, stderr] is relayed, stdout=1 a stderr=2
-    #         # as the field starting indexed is 0, so the +1 needs to be added
-    #         callback_linebuffered(read_fds.index(fd) + 1, lb)
+    # Process leftovers from line buffering
+    if encoding:
+        for (fd, lb) in linebufs.items():
+            if lb:
+                # [stdout, stderr] is relayed, stdout=1 a stderr=2
+                # as the field starting indexed is 0, so the +1 needs to be added
+                callback_linebuffered(read_fds.index(fd) + 1, lb)
```

Comment:
Still broken :)

```suggestion
                callback_linebuffered((fd, read_fds.index(fd) + 1), lb)
```

Reply:
Annyway, `red_fds.index(fd) + 1` is several times (2x seeing now). From the point that this should provide fd_type, what about rather write nested function `_get_fd_type`, like:
```
        def _get_fd_type(fd):
            return read_fds.index(fd) + 1
```